.github/workflows/test.yml

```
@@ -16,19 +16,8 @@ jobs:
       max-parallel: 5
       matrix:
         python-version: ['3.7', '3.8', '3.9', '3.10']
-        django-version: ['2.2', '3.1', '3.2', 'main']
-        drf-version: ['3.10', '3.11', '3.12']
-        exclude:
-          - python-version: '3.7'
-            django-version: 'main'
-          - python-version: '3.8'
-            django-version: 'main'
-          - django-version: '3.1'
-            drf-version: '3.10'
-          - python-version: '3.10'
-            django-version: '2.2'
-          - python-version: '3.10'
-            django-version: '3.1'
+        django-version: ['2.2', '3.1', '3.2', '4.0', 'main']
```

Please remove django 3.1
```suggestion
        django-version: ['2.2', '3.2', '4.0', 'main']
```